backend/models/points-model.js

```diff
@@ -0,0 +1,26 @@
+import mongoose from 'mongoose';
+
+const pointsSchema = new mongoose.Schema({
+  points: {
+    type: Number,
+    required: [true, "Points is required"],
+    min: [0, "Points cannot be negative"],
```

Comment:
Good point, unless we want to "punish" users for bad choices. 😉

Reply:
nah i think 0 is as bad as it should be haha

Reply:
haha, good one!